plugins/node/opentelemetry-plugin-mongodb/src/types.ts

```diff
@@ -43,7 +43,7 @@ export type MongoInternalTopology = {
 export enum MongodbCommandType {
   CREATE_INDEXES = 'createIndexes',
   FIND_AND_MODIFY = 'findAndModify',
-  IS_MASTER = 'isMaster',
+  IS_PRIMARY = 'isMaster',
```

Comment:
we could also rename the value here at its purely internal of the instrumentation